views/main/favorites.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Welcome</h1>
+
+<p>These are a few of my favorite things:</p>
+
+
+<!-- error msg: 
+# in main.rb
+class MyApp
+  get '/images/gita.jpg' do
+    "Hello World"
+  end
+end
+ -->
+<p>This is Gita. </p>
+<p><img src="../images/gita.jpg" width="500" height="400" alt="This is a picture of my cat, Gita."></p>
```

Comment:
You can make a folder in the project's root directory called **public** and put images there.

Then you can reference them with the `img` tag like `<img src="/gita.jpg"...`. Basically Sinatra takes everything in **public** and makes it available automatically from the root path. No controller actions needed to get things from the **public** directory.